### PR TITLE
Unapologetic Undead Buffs

### PR DESCRIPTION
**Pull request by Alberyk**
Halloween is close, and due to such, we must increase the spookyness.

This pr adds zombies. They spread via the bite, that will infect someone with a chemical, that will create a parasite, if left unchecked, it will turn the victim into a flesh eating undead.

### CODE REVIEW

**Comment by BotBOREALIS** [comment]
Icons with diff:
<details><summary>icons/mob/human_races/r_zombie.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
head||head|Created
head[1]||![head[1]](http://images.aurorastation.org/images/bf4d09d714f69692f10f6d4f583407cf.png)|Created
head[2]||![head[2]](http://images.aurorastation.org/images/4f425daba103aaba661aa7ff51f52853.png)|Created
head[3]||![head[3]](http://images.aurorastation.org/images/ef11fbb7096d0716f6c15cc51915b2f9.png)|Created
head_m||head_m|Created
head_m[1]||![head_m[1]](http://images.aurorastation.org/images/bf4d09d714f69692f10f6d4f583407cf.png)|Created
head_m[2]||![head_m[2]](http://images.aurorastation.org/images/4f425daba103aaba661aa7ff51f52853.png)|Created
head_m[3]||![head_m[3]](http://images.aurorastation.org/images/ef11fbb7096d0716f6c15cc51915b2f9.png)|Created
head_f||head_f|Created
head_f[1]||![head_f[1]](http://images.aurorastation.org/images/bf4d09d714f69692f10f6d4f583407cf.png)|Created
head_f[2]||![head_f[2]](http://images.aurorastation.org/images/4f425daba103aaba661aa7ff51f52853.png)|Created
head_f[3]||![head_f[3]](http://images.aurorastation.org/images/ef11fbb7096d0716f6c15cc51915b2f9.png)|Created
torso||torso|Created
torso[1]||![torso[1]](http://images.aurorastation.org/images/fda8515a406c3fd010eeffe676970497.png)|Created
torso[2]||![torso[2]](http://images.aurorastation.org/images/dfa2c17dfa02afe31591b658fc5fd8ed.png)|Created
torso[3]||![torso[3]](http://images.aurorastation.org/images/223c6ab1b6e3eaaa23d5928d400cd78c.png)|Created
torso_m||torso_m|Created
torso_m[1]||![torso_m[1]](http://images.aurorastation.org/images/fda8515a406c3fd010eeffe676970497.png)|Created
torso_m[2]||![torso_m[2]](http://images.aurorastation.org/images/dfa2c17dfa02afe31591b658fc5fd8ed.png)|Created
torso_m[3]||![torso_m[3]](http://images.aurorastation.org/images/223c6ab1b6e3eaaa23d5928d400cd78c.png)|Created
torso_f||torso_f|Created
torso_f[1]||![torso_f[1]](http://images.aurorastation.org/images/fda8515a406c3fd010eeffe676970497.png)|Created
torso_f[2]||![torso_f[2]](http://images.aurorastation.org/images/dfa2c17dfa02afe31591b658fc5fd8ed.png)|Created
torso_f[3]||![torso_f[3]](http://images.aurorastation.org/images/223c6ab1b6e3eaaa23d5928d400cd78c.png)|Created
groin||groin|Created
groin[1]||![groin[1]](http://images.aurorastation.org/images/5c367c8832ee46aa6aee48a032e19c2a.png)|Created
groin[2]||![groin[2]](http://images.aurorastation.org/images/aaaf66f2095a300aeb3640316c38182f.png)|Created
groin[3]||![groin[3]](http://images.aurorastation.org/images/562fa52c7a9ee405e3b12fd190315fc8.png)|Created
groin_m||groin_m|Created
groin_m[1]||![groin_m[1]](http://images.aurorastation.org/images/5c367c8832ee46aa6aee48a032e19c2a.png)|Created
groin_m[2]||![groin_m[2]](http://images.aurorastation.org/images/aaaf66f2095a300aeb3640316c38182f.png)|Created
groin_m[3]||![groin_m[3]](http://images.aurorastation.org/images/562fa52c7a9ee405e3b12fd190315fc8.png)|Created
groin_f||groin_f|Created
groin_f[1]||![groin_f[1]](http://images.aurorastation.org/images/5c367c8832ee46aa6aee48a032e19c2a.png)|Created
groin_f[2]||![groin_f[2]](http://images.aurorastation.org/images/aaaf66f2095a300aeb3640316c38182f.png)|Created
groin_f[3]||![groin_f[3]](http://images.aurorastation.org/images/562fa52c7a9ee405e3b12fd190315fc8.png)|Created
r_arm||r_arm|Created
r_arm[1]||![r_arm[1]](http://images.aurorastation.org/images/a8c8cda583783e0cee36a39f1a6a1dd1.png)|Created
r_arm[2]||![r_arm[2]](http://images.aurorastation.org/images/4f27e3ff41b0e61d34ca7152efcc439f.png)|Created
r_arm[3]||![r_arm[3]](http://images.aurorastation.org/images/6976942769e89f6e018d0c28782f20e4.png)|Created
l_arm||l_arm|Created
l_arm[1]||![l_arm[1]](http://images.aurorastation.org/images/3de61ccf541af4e17cffd7d5b199ff49.png)|Created
l_arm[2]||![l_arm[2]](http://images.aurorastation.org/images/6976942769e89f6e018d0c28782f20e4.png)|Created
l_arm[3]||![l_arm[3]](http://images.aurorastation.org/images/daa9d45096d91a369ee78a5b8ed45031.png)|Created
r_hand||r_hand|Created
r_hand[1]||![r_hand[1]](http://images.aurorastation.org/images/331be06ddc39f9e3c0e403b692c119e8.png)|Created
r_hand[2]||![r_hand[2]](http://images.aurorastation.org/images/fc53fac0f957308a2148d2e4737141de.png)|Created
r_hand[3]||![r_hand[3]](http://images.aurorastation.org/images/3ef7c936b106a1b1aebac19dce04a27a.png)|Created
l_hand||l_hand|Created
l_hand[1]||![l_hand[1]](http://images.aurorastation.org/images/3e2f8852d05350fc4bdb485220ebf122.png)|Created
l_hand[2]||![l_hand[2]](http://images.aurorastation.org/images/582eee0c6970e4acd695923be5113ea0.png)|Created
l_hand[3]||![l_hand[3]](http://images.aurorastation.org/images/af0b81d478005bcda50acfaa87274375.png)|Created
r_leg||r_leg|Created
r_leg[1]||![r_leg[1]](http://images.aurorastation.org/images/3abb27aa1bbf463027852ec2e1bd4abb.png)|Created
r_leg[2]||![r_leg[2]](http://images.aurorastation.org/images/69abaee993cdd77324f783c06cae06de.png)|Created
r_leg[3]||![r_leg[3]](http://images.aurorastation.org/images/c1a6314c9a2f207fbce746e17960a2f9.png)|Created
l_leg||l_leg|Created
l_leg[1]||![l_leg[1]](http://images.aurorastation.org/images/01e2532d4c1825f335bc29cab49cd11b.png)|Created
l_leg[2]||![l_leg[2]](http://images.aurorastation.org/images/69abaee993cdd77324f783c06cae06de.png)|Created
l_leg[3]||![l_leg[3]](http://images.aurorastation.org/images/60838ba1d3b22c8cc2be909e2aafd88a.png)|Created
r_foot||r_foot|Created
r_foot[1]||![r_foot[1]](http://images.aurorastation.org/images/5d46bfc69d1abba761983c2a3d7792aa.png)|Created
r_foot[2]||![r_foot[2]](http://images.aurorastation.org/images/4905fcad1137dd12d7220641bfe6ba9a.png)|Created
r_foot[3]||![r_foot[3]](http://images.aurorastation.org/images/0fa99a18e3ceb3725df0147a664b271d.png)|Created
l_foot||l_foot|Created
l_foot[1]||![l_foot[1]](http://images.aurorastation.org/images/d8baa03b43eade9d7a6d6fa27df5bbd3.png)|Created
l_foot[2]||![l_foot[2]](http://images.aurorastation.org/images/4905fcad1137dd12d7220641bfe6ba9a.png)|Created
l_foot[3]||![l_foot[3]](http://images.aurorastation.org/images/88d2d356e32d68abb6ffcfa6d2812a10.png)|Created
preview||preview|Created
</details>
<details><summary>icons/obj/surgery.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
blacktumor||blacktumor|Created
</details>

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm [hunk 1]
```diff
@@ -780,4 +780,26 @@
...
 	M.hallucination = max(M.hallucination, 50)
 	if(prob(10))
-		M.see_invisible = SEE_INVISIBLE_CULT
\ No newline at end of file
+		M.see_invisible = SEE_INVISIBLE_CULT
+
+/datum/reagent/toxin/trioxin
+	name = "Trioxin"
+	id = "trioxin"
+	description = "A synthetic compound of unknown origins, designated as a performance enhancing substance originally."
```
Comment (LordFowl) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm]: *A synthetic compound of unknown origins, designated originally as a performance enhancing substance.

File: code/modules/organs/subtypes/parasite.dm [hunk 2]
```diff
@@ -199,3 +201,59 @@
...
+	if(stage >= 3)
+		if(prob(10))
+			if(isundead(owner))
+				owner.adjustBruteLoss(-30)
+				owner.adjustFireLoss(-30)
+			else
+				owner << "<span class='cult'>You feel an insatiable hunger.</span>"
+				owner.nutrition = -1
+
+	if(stage >= 4)
```
Comment (skull132) [inline code/modules/organs/subtypes/parasite.dm]: Real quick. Are these stages supposed to stack, or is only one supposed to be active at any one time? At present, they stack. Ergo, someone with stage 4 or 5 will also have the effects from stages 2 - 3.

Comment (Alberyk) [reply code/modules/organs/subtypes/parasite.dm]: Yes, they are supossed to stack, in the same manner as the k'ois parasite does.

**Pull request merged**